From 8ed12e2301afd3c01a23ae4928822c0bf494e0be Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Mon, 6 Feb 2023 10:14:27 +0800
Subject: [PATCH 28/45] Revert "drm/amdgpu: Fix type of second parameter in
 trans_msg() callback"

This reverts commit a42a23bdae61a3aa789d1ad2da6d5583508262ca.
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_virt.h | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_virt.h b/drivers/gpu/drm/amd/amdgpu/amdgpu_virt.h
index fbd92fff8b06c..aea49bad914fa 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_virt.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_virt.h
@@ -62,8 +62,6 @@ struct amdgpu_vf_error_buffer {
 	uint64_t data[AMDGPU_VF_ERROR_ENTRY_SIZE];
 };
 
-enum idh_request;
-
 /**
  * struct amdgpu_virt_ops - amdgpu device virt operations
  */
@@ -73,8 +71,7 @@ struct amdgpu_virt_ops {
 	int (*req_init_data)(struct amdgpu_device *adev);
 	int (*reset_gpu)(struct amdgpu_device *adev);
 	int (*wait_reset)(struct amdgpu_device *adev);
-	void (*trans_msg)(struct amdgpu_device *adev, enum idh_request req,
-			  u32 data1, u32 data2, u32 data3);
+	void (*trans_msg)(struct amdgpu_device *adev, u32 req, u32 data1, u32 data2, u32 data3);
 };
 
 /*
-- 
2.25.1

